frontend/public/locales/fr/status.json

```diff
@@ -4,6 +4,7 @@
   "status-checker-content": "Il s'agit d'un outil qui vous permet de savoir où en est votre demande de participation au Régime canadien de soins dentaires",
   "online-status-heading": "Qui peut vérifier l'état de sa demande en ligne?",
   "online-status-content": "Si vous avez présenté une demande au titre du Régime canadien de soins dentaires, vous pouvez vérifier l'état de votre demande en ligne.",
+  "terms-conditions": "By using the Status Checker you agree to the following Terms and Conditions of Use and Privacy Notice Statement.",
```

Comment:
The French text is in the `/en/status.json` while the English text is in `/fr/status.json`